snuba/web/query.py

```diff
@@ -86,7 +86,7 @@ def _run_query_pipeline(
     ) and not request.settings.get_turbo():
         metrics.increment("sample_without_turbo", tags={"referrer": request.referrer})
 
-    extensions = dataset.get_extensions()
+    extensions = dataset.get_entity().get_extensions()
```

Comment:
Should dealing with this case be part of your entities plan or part of my joins plan ?

Reply:
I think it will be part of SnQL actually. SnQL Entity Support to me also includes parsing the entity(ies) out of the query and using them here.